source/help/getting-started/configuring-notifications.rst

```diff
@@ -1,124 +1,86 @@
 Configuring Notifications
 =========================
 
-Notifications in Mattermost alert you to unread messages and mentions.
+Desktop, email and mobile push notifications notify you of activity in Mattermost. When you join a team, Mattermost will notify you of messages directed at you, including when someone:
 
-Unreads and Mentions
-----------------------------------------
-
-Unread Messages Indicator
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The name of a channel in the left-hand sidebar is **bold** when
-there are unread messages in the channel. Clicking on the channel
-removes the bold indicator, and brings you to the most recent unread
-message in the channel.
-
-You can choose not to show unread indicators in a channel by clicking
-**Channel Menu** > **Notification Preferences** > **Mark Channel
-Unread** and selecting **Only for mentions**. This will only notify you if your name is mentioned or a
-keyword for which you're listening is entered.
+- Direct Messages you
+- Mentions your username or first name in a channel
```

Comment:
"first name" depends on the Words that trigger mentions setting.

Reply:
True, but by default this is on. I'm attempting to state the default settings that will trigger notifications which is "mentions and DMs"